setup.py

```diff
@@ -58,6 +58,6 @@ def get_version(project_dir=os.curdir):
 if __name__ == '__main__':
     os.environ['PBR_VERSION'] = get_version()
     setup(
-        setup_requires=['pbr'],
+        setup_requires=['pbr', 'dulwich'],
```

Comment:
Yep.
Actually its weird - I think ``pbr`` should pull it. I'll give it another try.